examples/sdk/python/basic-example/README.md

```diff
@@ -1,3 +1,15 @@
 # Basic Example
 
+Ideally, set up a Virtual Env with the Dagger Python SDK to run in:
+```
+python3 -m venv .venv
+source .venv/bin/activate
+pip install --upgrade pip; pip install --upgrade dagger-io
```

Comment:
```suggestion
pip install -U pip dagger-io
```
Upgrading pip isn't necessary, while good to do it's perhaps unnecessary extra for dagger specific instructions.